README.md

```diff
@@ -1,22 +1,35 @@
 # Internet-in-a-Box (IIAB)
 
-Welcome to the Git repository of the Internet-in-a-Box (IIAB) project. This is a community-based project developed and supported by volunteers from around the world. The Internet-in-a-Box (IIAB) is small, inexpensive device which provides essential Internet resources (like Wikipedia, OpenStreetMap, Khan Academy and others) without any Internet connection. It provides a local content server of the world’s Free Knowledge.
-
-The older version of this repository is at [github.com/xsce](http://github.com/xsce). We are using [Ansible](https://www.ansible.com) as the underlying technology to install, deploy, configure and manage the various software components.
+Welcome to the Git repository for Internet-in-a-Box (IIAB).  This community-based project is enabled by professional volunteers around the world.  Internet-in-a-Box (IIAB) is a small, inexpensive device which provides essential Internet resources (like Wikipedia, OpenStreetMap, Khan Academy and others) without any Internet connection.
```

Comment:
change to "including Wikipedia, OpenStreetMap, and Khan Academy" - the "like" and "and others" are redundant

Reply:
Thanks @aidan-fitz !

All suggestions incorporated here: https://github.com/iiab/iiab/pull/1211/commits/32b133ac91184cb7cbbdf6ac79b601db259bafc1